pallas-miniprotocols/src/common.rs

```diff
@@ -7,7 +7,7 @@ pub const TESTNET_MAGIC: u64 = 1097911063;
 pub const MAINNET_MAGIC: u64 = 764824073;
 
 /// A point within a chain
-#[derive(Clone)]
+#[derive(Clone, Eq, PartialEq)]
 pub struct Point(pub u64, pub Vec<u8>);
```

Comment:
what is the `Vec<u8>` here ? is it a block hash ?

Reply:
Yes, it is the hash. This struct was there before the introduction of pallas-crypto. We should turn it into a `Hash<>`.